bundles/org.openhab.binding.metofficedatahub/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,197 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="metofficedatahub"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="bridge">
+
+		<label>MetOffice DataHub Bridge</label>
+		<description>MetOffice DataHub API Bridge</description>
+
+		<properties>
+			<property name="Site Specific API Call Count"/>
+		</properties>
+
+		<config-description>
+			<parameter name="siteSpecificApiKey" type="text" required="true">
+				<context>password</context>
+				<label>Site Specific API Key</label>
+				<description>The API Key for the Site Specific subscription in your MET Office Data Hub account</description>
+			</parameter>
+			<parameter name="siteSpecificRateDailyLimit" type="integer" unit="s" min="1">
+				<label>API Daily Limit</label>
+				<description>A limit to the number of daily site specific API requests</description>
+				<default>250</default>
+				<advanced>true</advanced>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<thing-type id="siteSpecificApi">
+
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge"/>
+		</supported-bridge-type-refs>
+
+		<label>Forecast Data</label>
+		<description>Site Specific forecast data</description>
+
+		<channel-groups>
+			<channel-group id="commonResponseData" typeId="siteApiCommonDataGroup">
```

Comment:
Can you continue with this PR? 
According to the naming convention these id's should be `lower-case-hypen`
For reference: https://www.openhab.org/docs/developer/guidelines.html#naming-convention

Reply:
Just pushed the name updates, they are as minimal as I think I can make then and following the convention in terms of lower case hypens.

Reply:
still a few more to do looking through the source, but when the PR switches back from draft they should all be done.

Reply:
I think everything's ready cross checking through this PR, so its ready for another pass.